libcloud/test/compute/fixtures/gce/global_backendServices_web_service.json

```diff
@@ -7,14 +7,14 @@
  "backends": [
   {
    "description": "",
-   "group": "https://www.googleapis.com/resourceviews/v1beta1/projects/project_name/zones/us-central1-b/resourceViews/us-resources",
+   "group": "https://content.googleapis.com/compute/v1/projects/project_name/zones/us-central1-a/instanceGroups/myinstancegroup",
```

Comment:
Nope, you're right.  Should have been _www_.  I corrected this link and others I saw that were not correct in the fixtures that I created lately, but I think a fixture review/regeneration is in order, as we've talked about.